index.js

```diff
@@ -188,6 +205,14 @@ module.exports = {
             );
           }
         }
+        break;
+      }
+      case 'lighthouse.report': {
+        return this.storageManager.writeDataForUrl(
+          message.data,
+          'lighthouse',
```

Comment:
What about having different file endings here .json/.html so it easier to view?

Reply:
I thought about replacing this part with something like this:
```suggestion
      case 'lighthouse.report': {
        let reportExtenstion;
        switch (message.data[0]) {
          case '{': {
            reportExtenstion = ".json";
            break;
          };
          case 'c': {
            reportExtenstion = ".csv";
            break;
          };
          case '<': {
            reportExtenstion = ".html";
            break;
          };
          default: {
            reportExtenstion = "";
            break;
          }
        }
        return this.storageManager.writeDataForUrl(
          message.data,
          `lighthouse${reportExtenstion}`,
          message.url
        );
      }
```

Reply:
> 
> 
> What about having different file endings here .json/.html so it easier to view?
I thought about replacing it with something like this:
```
    case 'lighthouse.report': {
        let reportExtenstion;
        switch (message.data[0]) {
          case '{': {
            reportExtenstion = ".json";
            break;
          };
          case 'c': {
            reportExtenstion = ".csv";
            break;
          };
          case '<': {
            reportExtenstion = ".html";
            break;
          };
          default: {
            reportExtenstion = "";
            break;
          }
        }
        return this.storageManager.writeDataForUrl(
          message.data,
          `lighthouse${reportExtenstion}`,
          message.url
        );
      }
```

Reply:
Cool, do you have time to do it today? Or  we could just merge this first, and then fo follow to add more functionality later, that's fine for me. I will push a new release of sitespeed.io tonight, then we can include this PR.